install/prepare_configuration_payload.md

```diff
@@ -6,9 +6,9 @@ can be passed to the `csi` (Cray Site Init) program during the CSM installation
 Information gathered from a site survey is needed to feed into the CSM installation process, such as system name, system size, site network information for the CAN, site DNS configuration, site NTP configuration, network information for the node used to bootstrap the installation. More detailed component level information about the system hardware is encapsulated in the SHCD (Shasta Cabling Diagram), which is a spreadsheet prepared by HPE Cray Manufacturing to assemble the components of the system and connect appropriately labeled cables.
 
 How the configuration payload is prepared depends on whether this is a first time installation of CSM
-software on this system or the CSM software is being reinstalled. The reinstall scenario has the
+software on this system or the CSM software is being reinstalled. The *reinstall scenario* has the
 advantage of being able to use the configuration payload from the previous CSM installation
-and an extra configuration file which that installation generated.
+and an additional configuration file which that installation generated. The *first time* install scenario requires using the csi config tool to pass arguments to csi as well as the creation of a number of [Configuration Payload Files}(#configuration-payload-files).
```

Comment:
Second instance of CSI should be capitalized.

Reply:
Agreed, made the recommended change.